### ISSUE TITLE
Shortened query string param

### ISSUE THREAD

**Issue by gmanjames** [gmanjames/roboviz#23]
This is my bad for not noticing before approving changes. I noticed the strategy for shortening the path that we provide to a log-file in a query string parameter is making the assumption that it will be hosted on GitHub. I think Dr. Clark in our last meeting though said that a researcher should be allowed to host such files (log-files, assets, or otherwise) anywhere. This is probably fine for the demo tomorrow but if I remember correctly what Dr. Clark said we might have to rethink this a bit.

**Comment by kylefinter** [comment]
How would we go about finding out which file the user is wanting from where? Figuring out the path will be hard unless we add another parameter saying which source the file is coming from. Like add source of 'github' then we would prepend the url with the github link. and maybe for google drive we could have something similar.

**Comment by gmanjames** [comment]
Right, we'll have to brainstorm a bit. I think Dr. Clark mentioned some resource that we might mimic. Something 'flow' maybe? We will have to ask him.

**Comment by anthonyjclark** [comment]
[Hub](https://github.com/github/hub)
